app/views/languages/_about.erb

```diff
@@ -5,7 +5,7 @@
   If you've downloaded the <a href='/cli'>command-line client</a> and have <%= track.language %> installed
   on your machine, then go ahead and fetch the first problem.
 </p>
-<%= syntax "exercism #{track.id} #{track.problems.first.slug}", "plain" %>
+<%= syntax "exercism fetch #{track.id} #{track.problems.first.slug}", "plain" %>
```

Comment:
If we take out the `#{track.problems.first.slug}` it will leave it as vanilla as possible, as it will fetch the first problem not done, which will be the first one?